tests/features/transactions.go

```diff
@@ -1,62 +1,41 @@
 package features
 
 import (
+	"context"
 	"fmt"
-	"regexp"
-	"strconv"
-	"strings"
 	"time"
+
+	kowala "github.com/kowala-tech/kcoin"
+	"github.com/kowala-tech/kcoin/accounts"
+	"github.com/kowala-tech/kcoin/core/types"
 )
 
-var txRegexp = regexp.MustCompile(`0x[0-9a-f]{64}`)
+func (ctx *Context) ITransferKUSD(kcoin int64, from, to string) error {
 
-func (context *Context) ITransferKUSD(kcoin int, from, to string) error {
-	command := fmt.Sprintf(
-		`
-			personal.unlockAccount(eth.coinbase, "test");
-			eth.sendTransaction({from:eth.coinbase, to: "%s", value: web3.toWei(%v, 'ether')})
-		`,
-		context.accountsCoinbase[to],
-		kcoin)
-	res, err := context.cluster.Exec(context.accountsNodeNames[from], command)
+	tx, err := ctx.sendFunds(ctx.accounts[from], ctx.accounts[to], kcoin)
 	if err != nil {
 		return err
 	}
-	if !txRegexp.MatchString(res.StdOut) {
-		return fmt.Errorf("Expected transaction, received: %v", res.StdOut)
-	}
-	err = waitFor("transaction in the blockhain", 1*time.Second, 5*time.Second, func() bool {
-		isInBlockchain, err := context.isTransactionInBlockchain(res.StdOut)
+
+	return waitFor("transaction in the blockhain", 1*time.Second, 5*time.Second, func() bool {
+		isInBlockchain, err := ctx.isTransactionInBlockchain(tx)
 		return err == nil && isInBlockchain
 	})
-	if err != nil {
-		return err
-	}
-	return nil
 }
 
-func (context *Context) ITryTransferKUSD(kcoin int, from, to string) error {
-	command := fmt.Sprintf(
-		`
-			personal.unlockAccount(eth.coinbase, "test");
-			eth.sendTransaction({from:eth.coinbase, to: "%s", value: web3.toWei(%v, 'ether')})
-		`,
-		context.accountsCoinbase[to],
-		kcoin)
-	res, err := context.cluster.Exec(context.accountsNodeNames[from], command)
-	if err != nil {
-		return err
-	}
-	context.lastTxStdout = res.StdOut
+func (ctx *Context) ITryTransferKUSD(kcoin int64, from, to string) error {
```

Comment:
err is always nil now

Reply:
Yes, but this is the step implementation. I'd prefer to keep it consistent with the rest, and return `error` even though right now it's always nil. Would you prefer to not return anything?

Reply:
Actually, godog handlers need to return `error`

Reply:
Just notices, does need the err, it's godog method signature